src/flint/nmod.jl

```diff
@@ -355,6 +355,15 @@ function zero!(z::zzModRingElem)
   return zzModRingElem(UInt(0), R)
 end
 
+function one!(z::zzModRingElem)
+  R = parent(z)
+  return zzModRingElem(UInt(1), R)
```

Comment:
Actually, this one is wrong (as the conformance tests show), and should be
```suggestion
  return zzModRingElem(UInt(R.n == 1 ? 0 : 1), R)
```

But really even better is to just drop this and rely on the default implementation.

Reply:
I am in favor of dropping this, but all of the other "dump" implementation right above and below this as well. They are identical to the default, and bring no improvement whatsoever

Reply:
Yeah I have that locally already but it revealed another bug in AA. PRs will come after lunch.